### PR TITLE
chore: add prlint rule

### PR DESCRIPTION
**Pull request by feng-tao**
### Summary of Changes

Add prlint rule. Starts with simple pr title check.
Other available rules could be found in https://github.com/apps/prlint

This is for https://github.com/lyft/amundsen/issues/459.

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely.
- [x] PR includes a summary of changes.
- [x] My commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"

### CODE REVIEW

**Comment by feng-tao** [comment]
If the rule looks good, I will add it for all the other repos and share with the slack community.

File: .github/prlint.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "title": [
+    {
+      "pattern": "^(build|ci|docs|feat|fix|perf|refactor|style|test|chore|other)((.+))?:\\s.+",
+      "message": "Your title needs to be prefixed with a topic."
+    }
+  ]
```
Comment (Golodhros) [inline .github/prlint.json:7]:
```suggestion 
  ],
  "body": [
        {
            "pattern": ".{1,}",
            "message": "You need to have some content in the description."
        }
    ],
```

File: .github/prlint.json [hunk 2]
```diff
@@ -4,5 +4,11 @@
...
       "message": "Your title needs to be prefixed with a topic."
     }
-  ]
+  ],
+  "body": [
+        {
+            "pattern": ".{1,}",
+            "message": "You need to have some content in the description."
+        }
+    ],
```
Comment (Golodhros) [inline .github/prlint.json:13]:
```suggestion
  ]
```
Sorry, this would break the JSON format!

Comment (feng-tao) [reply .github/prlint.json]: let me test it out a bit.

**Comment by Golodhros** [comment]
Related to this, here is my proposal for labels and PR titles:
https://docs.google.com/document/d/1dTY-W0vGAX5Oe68Zbq6UIy5QWCoJK8FRu1ibxcRAI_M/edit#heading=h.85m4a2v9jj3e

**Comment by feng-tao** [comment]
cc @jinhyukchang @ttannis @Golodhros 
this pr is merged now.

**Pull request merged**